cdap-docs/integrations/source/partners/cloudera/index.rst

```diff
@@ -56,6 +60,12 @@ results.
 
 - |cloudera-ingesting|_ Ingest and explore data using **Impala**
 
+.. |cloudera-navigator| replace:: **Navigator Integration:**
+.. _cloudera-navigator: navigator-integration.html
+
+- |cloudera-navigator|_ Metadata discovery and search using CDAP's **Navigator Integration
```

Comment:
Because the title of the linked page is "Navigator Integration”, I would prefer to keep that as the link, and use:

​**Navigator Integration:**​ Metadata search and discovery of CDAP entities from **Cloudera Navigator**

as we usually have that the link text matches as closely as possible text found on the landing page. Of course, we ​_could_​ change the title of the landing page...